### ISSUE TITLE
Add Ubuntu Artful

### ISSUE THREAD

**Issue by jsoref** [certbot/website#299]
Ubuntu Zesty has reached End Of Life ->
https://certbot.eff.org/all-instructions/#ubuntu-17-04-zesty-apache

Ubuntu Artful should be listed.

```certbot-ubuntu-certbot-artful.list
deb http://ppa.launchpad.net/certbot/certbot/ubuntu artful main
# deb-src http://ppa.launchpad.net/certbot/certbot/ubuntu artful main
```

**Comment by SwartzCr** [comment]
@jsoref any chance you want to make a PR for this?

**Comment by jsoref** [comment]
Yes, although not immediately

**Comment by bmw** [comment]
Fixed in #308.
